### PR TITLE
Improve readability and consistency in create_dataframes.py

### PR DESCRIPTION
**Pull request by vansika**
This PR formats create_dataframe.py for improving consistency.

I have also deleted the associated HTML files since some of the information it contains is stored in dataframe_metadata and some of the information is sent over in a message over rabbitMQ to lemmy which is then mailed using observability mailer.

### CODE REVIEW

File: listenbrainz_spark/recommendations/create_dataframes.py [hunk 1]
```diff
@@ -160,30 +158,36 @@ def get_listens_for_training_model_window(to_date, from_date, metadata, dest_pat
...
-            (partial_listens_df.artist_msid == recording_artist_mapping_df.msb_artist_msid),
-        'inner'
-    )
+    condition = [
+        partial_listens_df.recording_msid == msid_mbid_mapping_df.msb_recording_msid,
+        partial_listens_df.artist_msid == msid_mbid_mapping_df.msb_artist_msid
+    ]
+
+    df = partial_listens_df.join(msid_mbid_mapping_df, condition, 'inner')
     # msb_release_name_matchable is skipped till the bug in mapping is resolved.
```
Comment (mayhem) [inline listenbrainz_spark/recommendations/create_dataframes.py:178]: This bug should now be fixed. Can you please confirm?

Comment (vansika) [reply listenbrainz_spark/recommendations/create_dataframes.py:178]: I don't have the latest mapping on my local machine (and I cannot download for the size) :( 

I will ask iliekcomputers to run a test query for this. But if you have removed the bug I can remove the comment.

Comment (mayhem) [reply listenbrainz_spark/recommendations/create_dataframes.py:178]: You could download the mapping into your account on newleader and check there....

Comment (vansika) [reply listenbrainz_spark/recommendations/create_dataframes.py:178]: Oh like check manually? Cool, will do.

Comment (vansika) [reply listenbrainz_spark/recommendations/create_dataframes.py:178]: should I keep the comment and merge this PR since we found a few bugs yesterday?

Comment (mayhem) [reply listenbrainz_spark/recommendations/create_dataframes.py:178]: I pushed a new mapping. Have you verified that the new mapping contains further bugs?

**Pull request merged**